cmd/nerdctl/container/container_run_network_linux_test.go

```diff
@@ -504,6 +505,37 @@ func TestSharedNetworkStack(t *testing.T) {
 		AssertOutContains(testutil.NginxAlpineIndexHTMLSnippet)
 }
 
+func TestSharedNetworkWithNone(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("--network=container:<container name|id> only supports linux now")
+	}
+	base := testutil.NewBase(t)
+
+	containerName := testutil.Identifier(t)
+	defer base.Cmd("rm", "-f", containerName).AssertOK()
```

Comment:
Generally speaking:
- favor t.Cleanup to defer
- do also run your cleanup routine before the test itself, so that local repeat runs do not fail if there are leftover from prior runs

Alternatively, you may want to use the new [test framework](https://github.com/containerd/nerdctl/blob/main/docs/testing/tools.md) that is designed to take care of these.

Reply:
it is just prior to the run for the statement for running the continous container. (sleep infinity)